cabby/geo/walk.py

```diff
@@ -107,23 +146,19 @@ def get_start_poi(
     A single POI.
   '''
 
-  dist = map.poi['centroid'].apply(
-    lambda x: util.get_distance_km(end_point['geometry'], x))
-
   # Get closest nodes to points.
-  dest = ox.get_nearest_node(
-    map.nx_graph, util.tuple_from_point(end_point['centroid']))
+  dest_osmid = end_point['osmid']
 
   # Find nodes whithin 2000 meter path distance.
   outer_circle_graph = ox.truncate.truncate_graph_dist(
-    map.nx_graph, dest, max_dist=2000, weight='length')
+    map.nx_graph, dest_osmid, max_dist=2000+POI_DISTANCE, weight='length')
 
   outer_circle_graph_osmid = list(outer_circle_graph.nodes.keys())
 
   try:
-    # Get graph that is too close (less than 400 meter path distance)
+    # Get graph that is too close (less than 200 meter path distance)
     inner_circle_graph = ox.truncate.truncate_graph_dist(
-      map.nx_graph, dest, max_dist=400, weight='length')
+      map.nx_graph, dest_osmid, max_dist=200+POI_DISTANCE, weight='length')
```

Comment:
Should the number `200` be configurable?

Reply:
Changed to MAX_PATH_DIST and MIN_PATH_DIST